html/images/os/sagemcom.svg

```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is more of a logo than an icon.  It also needs to be minimized.
https://docs.librenms.org/Developing/os/Initial-Detection/#icon-and-logo

Reply:
changed